libs/cardano-ledger-core/src/Cardano/Ledger/Address.hs

```diff
@@ -193,20 +211,29 @@ addrPtrNormalize = \case
   addr -> addr
 
 -- | An account based address for rewards
-data RewardAcnt c = RewardAcnt
-  { getRwdNetwork :: !Network
-  , getRwdCred :: !(Credential 'Staking c)
+data RewardAccount c = RewardAccount
+  { raNetwork :: !Network
+  , raCredential :: !(Credential 'Staking c)
   }
   deriving (Show, Eq, Generic, Ord, NFData, ToJSONKey, FromJSONKey)
 
+pattern RewardAcnt :: Network -> Credential 'Staking c -> RewardAccount c
+pattern RewardAcnt a b = RewardAccount a b
```

Comment:
I believe you can do something like this:
```suggestion
pattern RewardAcnt {getRwdNetwork,getRwdCred} = RewardAccount getRwdNetwork getRwdCred

{-# DEPRECATE getRwdNetwork "In favor of `raNetwork`" #-}
{-# DEPRECATE getRwdCred "In favor of `raCredential`" #-}
```

This will allow even for record updates to still work, eg. `ra { getRwdNetwork = foo }`

Reply:
Ah yes. Done, cheers! :+1: